exchange.js

```diff
@@ -21,9 +25,18 @@ CLEAR_BUTTON.addEventListener("click", clearCurrencies);
 fetch(url)
   .then((response) => response.json())
   .then((data) => {
-    usdData = data.rates.USD;
-    rsdData = data.rates.RSD;
-    console.log("Курсы валют получены:", { USD: usdData, RSD: rsdData });
+    rates = {
+      RSD: data.rates.RSD,
+      USD: data.rates.USD,
+      EUR: data.rates.EUR,
+    };
+    console.log("Курсы валют получены:", rates);
```

Comment:
let's get rid of console log calls, they shouldn't be in production without good reason